masterlist.yaml

```
@@ -29890,6 +29903,11 @@ plugins:
       - <<: *useInstead
         subs: [ '[lilebonymace''s patches and xEdit scripts](https://www.nexusmods.com/skyrimspecialedition/mods/36042/) (**Open World Loot - C.O.I.N.esp**)' ]
 
+  - name: 'Sofia - RDO Patch.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/70950' ]
+    # Missing RDO as a master
+    after: [ 'Relationship Dialogue Overhaul.esp' ]
```

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/70950/'
        name: 'Sofia Bug and Patch Hub'
    req: [ 'Relationship Dialogue Overhaul.esp' ]
```

Requirements are better for missing masters, they load the plugin after & display an error message if the required plugin is missing. Also, some tweaks to the URL for conformity & convenience.

---

Oh sweet. Thanks! I'll do better xD